src/libServer/APIServer.h

```diff
@@ -50,6 +54,9 @@ class APIServer {
     /// Limit in bytes for POST bodies of incoming requests (security)
     size_t inputBodyLimit = 5 * 1024 * 1024;
```

Comment:
A suggestion: use the unit as part of the name (more readable and saves the developer the need to look up the comments): _limitBodyLimitBytes_